index.html

```diff
@@ -38,7 +38,7 @@ <h1>Random Number Generator</h1>
             } else {
                 // Use a random fact when not divisible by 13
                 var randomFact = getRandomFact();
-                document.getElementById('randomn').innerHTML = "..You lost! See the Fun Fact: <br> " + randomFact;
+                document.getElementById('randomn').innerHTML = ".See the Fun Fact: <br> " + randomFact;
```

Comment:
Consider using template literals for better readability and easier manipulation of the strings. [medium]